lib/gui/app.js

```diff
@@ -298,11 +298,18 @@ app.controller('HeaderController', function(SelectionStateModel, OSOpenExternalS
 
 });
 
-app.controller('StateController', function($state) {
+app.controller('StateController', function($rootScope, $state) {
 
   /**
    * @param {string} state - state page
    */
   this.is = $state.is;
 
+  /**
+   * @param {function} func - function to pass $rootScope to
```

Comment:
This is missing the `@summary`, `@public`, `@example`, etc. Also `function` should be `Function`